### ISSUE TITLE
How To Setup

### ISSUE THREAD

**Issue by Danny-johziy** [Tecnophille/NFT-Buy-Sell-Marketplace#5]
please can you provide me a guide how to host this project

**Comment by Tecnophille** [comment]
You need LAMP/XAMP/WAMP to test it locally before uploading to live server.

**Comment by Danny-johziy** [comment]
Yes am running it on my XAMP but I have a problem, a user is unable to register it's just showing up some error

**Comment by Danny-johziy** [comment]
![Screenshot_20230928-205325](https://github.com/Tecnophille/NFT-Buy-Sell-Marketplace/assets/62003376/696137e5-c6c7-474c-84dd-8f9ca2d7f479)

**Comment by Danny-johziy** [comment]
This is the error I get whenever I try to sign in

**Comment by Tecnophille** [comment]
Try debug to resolve this issue.

**Comment by Danny-johziy** [comment]
Am still learning PHP Laravel so that's why am having some difficulty debugging it, can you please help me.

**Comment by Tecnophille** [comment]
chatGPT can assist you with debugging.

**Comment by kiomarytech** [comment]
Please i am new here, i am trying to upload on my cpanel server but it is now installing. Please i need guidance

**Comment by Tecnophille** [comment]
Have you hosted it first locally?

**Comment by ashujain108** [comment]
Hi @Tecnophille ,
I was trying to setup on my local machine.
could you tell me the php and laravel version which is used for this?
as my debugging say php version 7.2 so i was using 7.2.34.
but laravel version could not get debug.
Tried to install  with 'composer install' command but getting some error.

please help me to setup on my local machine.

Thanks
Ashu Jain

**Comment by ashujain108** [comment]
@Danny-johziy can you help me on this.
